README.md

```diff
@@ -1,6 +1,14 @@
 # Paraview-doc
 
-Automated publication of ParaView documentation via github pages.
+Automated publication of ParaView documentation via GitHub pages.
+
+Given Paraview source and build trees, tt provides the infrastructure for
+* generating the associated ParaView C++ and Python API documentation
+* automatically updating the site available at https://kitware.github.io/paraview-docs/ after changes are pushed onto the `gh-pages` branch
+
+The updates to the `gh-pages` branch are pushed every night by the official "dashboard" build & testing ParaView and maintained by [@Kitware](https://github.com/Kitware).
```

Comment:
```suggestion
The updates to the `gh-pages` branch are pushed every night by the official "dashboard" building & testing ParaView and maintained by [@Kitware](https://github.com/Kitware).
```

Reply:
This sentence is odd, maybe:

The updates to the `gh-pages` branch are pushed every night by the official ParaView CI infrastructure which is maintained by [@Kitware](https://github.com/Kitware).